Ports/chibi-scheme/package.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='chibi-scheme'
+version='0.10'
+useconfigure='false'
+files=(
+    "https://github.com/ashinn/chibi-scheme/archive/refs/tags/${version}.tar.gz ae1d2057138b7f438f01bfb1e072799105faeea1de0ab3cc10860adf373993b3"
```

Comment:
This Port looks reasonable to me, but we recently merged a change to the files syntax. I believe all this needs is a `#` b/w the .tar.gz and the sha256 hash to be immediately workable.